packages/kotti-ui/source/kotti-field-select/styles.scss

```diff
@@ -29,6 +29,25 @@
 		}
 	}
 
+	&-actions {
```

Comment:
don't forget to change this if you follow BEM, likewise for any other class accessors below